src/components/TrafimageMaps/TrafimageMaps.js

```diff
@@ -160,6 +161,28 @@ class TrafimageMaps extends React.PureComponent {
      * @private
      */
     this.store = getStore();
+
+    this.theme = createMuiTheme({
```

Comment:
Is this required for the optional `TopicMenuBottom` component? If so I would prefer wrapping just `TopicMenuBottom` in `TopicMenu` with the provider or at least have a comment here. Otherwise we'll forget what this is about.